G/GMT/build_tarballs.jl

```diff
@@ -0,0 +1,103 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "GMT"
+version = v"6.4.0"
+
+GSHHG_VERSION="2.3.7"
+DCW_VERSION="2.1.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/GenericMappingTools/gmt", 
+    "19413d486888ebdc3c01aeed3707bcc88eb727df"),
+
+    ArchiveSource("https://github.com/GenericMappingTools/gshhg-gmt/releases/download/$GSHHG_VERSION/gshhg-gmt-$GSHHG_VERSION.tar.gz",
+        "9bb1a956fca0718c083bef842e625797535a00ce81f175df08b042c2a92cfe7f"),
+
+    ArchiveSource("https://github.com/GenericMappingTools/dcw-gmt/releases/download/$DCW_VERSION/dcw-gmt-$DCW_VERSION.tar.gz",
+        "d4e208dca88fbf42cba1bb440fbd96ea2f932185c86001f327ed0c7b65d27af1")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+
+GSHHG_VERSION_numeric=2.3.7     
+GSSHG="gshhg-gmt-2.3.7"
+DCW="dcw-gmt-2.1.1"
+EXT="tar.gz"
+
+cd $WORKSPACE/srcdir
+cd gmt
+mkdir build
+cd build/
+
+cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DHAVE_QSORT_R_GLIBC=False \
+    -DHAVE___BUILTIN_BSWAP16=False \
+    -DHAVE___BUILTIN_BSWAP32=False \
+    -DHAVE___BUILTIN_BSWAP64=False \
+    -DGMT_ENABLE_OPENMP=True \
+    -DGSHHG_PATH=/workspace/srcdir/${GSSHG} \
+    -DGSHHG_VERSION=${GSHHG_VERSION_numeric} \
+    -DDCW_PATH=/workspace/srcdir/${DCW} \
+    .. 
+make -j${nproc} 
+make install 
+
+if [[ "${target}" == *-mingw* ]]; then
+    install -Dvm 755 /workspace/destdir/bin/gmt.${dlext} "${libdir}/libgmt.${dlext}"
```

Comment:
Please, ***never*** reference `/workspace/destdir` directly, use the most appropriate [environment variable](https://docs.binarybuilder.org/stable/build_tips/#Automatic-environment-variables), otherwise this makes us to reorganise the structure of the filesystem if needed in the future.

Reply:
I'm also not sure why you're moving these files in the first place.